Automattic/woocommerce-payments
# Fix for issue where border displayed on MultiCurrency block widget even when disabled

Fixes #3044 

#### Changes proposed in this Pull Request









- Fix for an issue where the border for Multi Currency block widget would not respect the 'border' property and would display regardless.

#### Testing instructions





* Go to Appearance -> Widgets in WP Admin and add the `Currency Switcher Block` widget somewhere on your site (if you haven't already).
* Click on the widget to open the settings pane on the right hand side of the screen, change the `Border` option to off (uncheck it), and then click `Update` in the top right corner of the screen.
* Go to your store, verify you can see the currency switcher block widget and the border does *not* display.
* Go back to the settings, turn the border back on, save the change.
* Go back to the store again and verify the border displays when the setting is checked.

With border:

<img width="154" alt="Screenshot 2021-10-11 at 16 06 27" src="https://user-images.githubusercontent.com/11770181/136813427-f4619dcd-99b9-42ae-b654-cdfea6fef867.png">


Without border:

<img width="219" alt="Screenshot 2021-10-11 at 16 06 33" src="https://user-images.githubusercontent.com/11770181/136813434-23a4a65c-b7e3-4606-be9e-d97e0256f8a3.png">

(Note that the actual appearance may vary depending on other settings.)

-------------------


**Post merge**
